### PR TITLE
[web]allow editable list to be readonly

### PR DESCRIPTION
**Pull request by mba-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

Task : https://www.odoo.com/web#id=32612&view_type=form&model=project.task&action=327&menu_id=4720
Pad : https://pad.odoo.com/p/r.e1cf1fd65c3cd171bdf8ad2bb4c83d20
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by msh-odoo** [comment]
LGTM

**Comment by jem-odoo** [comment]
LGTM.
But can you update commit message: explain why we need this change, what are the consequences. Also, add the task-id.

**Comment by jem-odoo** [comment]
@ged-odoo can I have you input on this ? 
Many thanks.

File: addons/web/static/src/js/views/list/list_editable_renderer.js [hunk 1]
```diff
@@ -453,6 +453,15 @@ ListRenderer.include({
...
         }
     },
+    /**
+     * Check whether list view is ediable listview
+     * consider editable if editable attribute given and edit=true
+     *
+     * @private
+     */
+    _isEditable: function () {
+        return this.editable && (!this.arch.attrs.edit || !!JSON.parse(this.arch.attrs.edit));
```
Comment (aab-odoo) [inline addons/web/static/src/js/views/list/list_editable_renderer.js]: this should be directly encoded in the 'editable' param, so you should move this logic to list_view.js (and use this.controller.Params.activeActions.edit)

File: addons/web/static/tests/views/list_tests.js [hunk 2]
```diff
@@ -168,6 +168,25 @@ QUnit.module('Views', {
...
+            model: 'foo',
+            data: this.data,
+            viewOptions: {hasSidebar: true},
+            arch: '<tree editable="top" edit="0"><field name="foo"/></tree>',
+        });
+
+        assert.ok(list.$('tbody td.o_list_record_selector').length, 'should have at least one record');
+
+        await testUtils.dom.click(list.$('tr td:not(.o_list_record_selector)').first());
+        assert.strictEqual(list.$('tbody tr.o_selected_row').length, 0, "should not have editable row");
```
Comment (aab-odoo) [inline addons/web/static/tests/views/list_tests.js]: you can now use assert.containsNone

**Comment by jem-odoo** [comment]
As Ged suggest, can you add something like 
` .. note:: if the ``edit`` attribute is set to ``false``, the ``editable`` option will be ignored. ` in the view list documentation, in the 'editable' attribute section.
thanks

**Comment by jem-odoo** [comment]
Can you squash the 2 commits please ? Only one will be enough. Thanks

**Comment by jem-odoo** [comment]
@robodoo r+

**Comment by jem-odoo** [comment]
@robodoo r+

**Comment by jem-odoo** [comment]
@robodoo r+

**Comment by jem-odoo** [comment]
@robodoo r+

**Pull request closed without merging**